style.css

```diff
@@ -384,4 +384,78 @@ body {
     text-align: left;
     color: #344563;
   }
+
+  footer {
+    background-color: #6070ff;
+    padding-top: 103px;
+    border-radius: 100px 0 0 0;
+    background-image: url(./figmaImages/images/contactformshapes-mobile.png);
+    background-repeat: no-repeat;
+    background-position: 100% 20px;
+  }
+
+  .form {
+    width: 87%;
+    margin: auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+  }
+
+  .form h2 {
+    color: #fff;
+    font-weight: bold;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .form p {
+    margin-top: 12px;
+    color: #ebebff;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  .contact-form {
+    width: 100%;
+    padding: 45px 0 0 0;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .contact-form input {
+    width: 100%;
+    margin-bottom: 20px;
+    padding: 15px 38px 15px 16px;
+    border-radius: 8px;
+    border: 1px solid #cfd8dc;
+    color: #172b4d;
+    font-weight: normal;
+    font-size: 17px;
+    line-height: 20px;
+  }
+
+  .contact-form textarea {
+    width: 100%;
+    min-width: 100%;
+    max-width: 100%;
+    margin-bottom: 20px;
+    padding: 15px 38px 15px 16px;
+    border-radius: 8px;
+    border: 1px solid #cfd8dc;
+    color: #b3bac5;
+    font-weight: normal;
+    font-size: 15px;
+    line-height: 24px;
+    min-height: 170px;
+  }
+
+  .formSpace button {
+    margin: 0;
+    padding: 12px 16px;
+    width: auto;
+    border-radius: 8px;
```

Comment:
Nice work so far, form submitted successfully. However, there are some few differences between your design and the actual one as shown below: 

- [ ] The margins around the form inputs are different from the actual design.
- [ ] The submit button looks different from the actual design. 
- [ ] Button interactions such as hover, active (when it's pressed), disabled and enable (default state) are not implemented.
Kindly check and update your design with the following observations so that your project will look more professional and match the design. :+1:  

### Your design 

![Screenshot from 2023-07-13 21-50-06](https://github.com/gilberthappi/portfolio/assets/65233860/75e19975-8b8d-483f-bf28-6b42ccc3e9ae)



### Actual design

![Screenshot from 2023-07-13 21-52-31](https://github.com/gilberthappi/portfolio/assets/65233860/44ad869d-7e61-4486-8b7f-285ad7dd24f0)